src/main.js

```diff
@@ -275,11 +275,16 @@ function resizeHandlerY(e){
   const mouseY = e.pageY;
   const main = document.querySelector('main').getBoundingClientRect();
   const topHeight = mouseY - main.top;
+  const mainHeight = main.height;
 
+  // set the height for main
   document.querySelector('main').style.height = topHeight + 'px';
-  document.querySelector('.bottom-part').style.height = 100% - (topHeight + 'px');
 
-  const change = window.innerHeight - mouseY - 25 + '%'
+  // set the height for the whole bottom div
+  document.querySelector('.bottom-part').style.height = (mainHeight - topHeight) + 'px';
```

Comment:
This is a good change. Will you also set the initial height so there isn't a "blue margin" under the output box before the first time you resize?

Reply:
I'd vote that we avoid fixing the size of content as much as possible (but don't go crazy). I think it'd be more sustainable to use Flexbox or Grid to lay out the view. They'd make it easier to toggle the visibility of certain panels. The mouse drag should change the flex-grow property or the row or column's weight in the grid-template-* property.